wrappers/python/simtk/openmm/__init__.py

```diff
@@ -20,6 +20,8 @@
 
 from simtk.openmm.openmm import *
 from simtk.openmm.vec3 import Vec3
+from mtsintegrator import MTSIntegrator
+from amd import AMDIntegrator, AMDForceGroupIntegrator, DualAMDIntegrator
```

Comment:
Ah, OK, this is something `2to3` fixes (indeed, as-is implicit relative imports raise an exception in Python 3).

There's a little concern that if anybody had downloaded these integrators previously and have them lying around somewhere in `sys.path`, this import will mistakenly pick them up from there rather than this package.